initialscreen.php

```diff
@@ -0,0 +1,256 @@
+<?php
+require 'dbConn.php';
+function retrieve_data($db)
+{
+    $retrieve_statement = $db->prepare('SELECT `content`, `picture_link`, `phone_number`, `email` FROM `about_me` WHERE `id` = 1;');
+    $retrieve_statement->execute();
+    $start_data = $retrieve_statement->fetch();
+    return $start_data;
+}
+
+$start_data = retrieve_data($db);
+
+function retrieve_data_project($db, $selector) {
+
+    $retrieve_statement = $db->prepare('SELECT `picture_link`, `picture_text`, `picture_image_front`, `code_link` FROM `projects` WHERE `id` = ?;');
```

Comment:
if you dont set an ID and use a fetchAll, you can get all 9 in one function call